pkg/remote/aws/repository/s3_repository.go

```diff
@@ -17,7 +17,7 @@ type S3Repository interface {
 	ListBucketInventoryConfigurations(bucket *s3.Bucket, region string) ([]*s3.InventoryConfiguration, error)
 	ListBucketMetricsConfigurations(bucket *s3.Bucket, region string) ([]*s3.MetricsConfiguration, error)
 	ListBucketAnalyticsConfigurations(bucket *s3.Bucket, region string) ([]*s3.AnalyticsConfiguration, error)
-	GetBucketLocation(bucket *s3.Bucket) (string, error)
+	GetBucketLocation(bucketName string) (string, error)
```

Comment:
Is it really needed to do it in that PR ?

Reply:
Yeah this is a mandatory side effect.

Reply:
(We don't want to pass a `s3.Bucket` object in attributes)